store/iavlstore_test.go

```diff
@@ -73,13 +73,70 @@ func TestIAVLIterator(t *testing.T) {
 	iavlStore := newIAVLStore(tree, numHistory)
 	iter := iavlStore.Iterator([]byte("aloha"), []byte("hellz"))
 	expected := []string{"aloha", "hello"}
-	for i := 0; iter.Valid(); iter.Next() {
+	i := 0
+	for i = 0; iter.Valid(); iter.Next() {
```

Comment:
replace with 
```
for ; iter.Valid(); iter.Next() {
```

Reply:
Want it to be the same as other loops below, so I'll just declare i above without initializing.